#!/usr/bin/env python
PACKAGE = "dynamic_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_lower", int_t, 0, "h_lower", 100,  0, 180)
gen.add("s_lower", int_t, 0, "s_lower", 60, 0, 255)
gen.add("v_lower", int_t, 0, "v_lower", 60, 0, 255)
gen.add("h_upper", int_t, 0, "h_upper", 110, 0, 180)
gen.add("s_upper", int_t, 0, "s_upper", 255, 0, 255)
gen.add("v_upper", int_t, 0, "v_upper", 255, 0, 255)

exit(gen.generate(PACKAGE, "dynamic_tutorials", "Tutorials"))
